Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/bpmneditor/editor/css/theme_norm.css

```diff
@@ -77,7 +77,7 @@ table-layout:fixed;
 	width:10px;
 }
 
-#oryx_editor_header {
+#Apromore_editor_header {
        background-image: url('../images/header_bg.small.gif');
```

Comment:
It's not critical, but css selectors commonly use kebab-case. Here we use a mix of CamelCase and snake_case.